charts/kuberpult/Makefile

```diff
@@ -25,40 +25,42 @@ REL_DIR := $(subst $(TOP_DIR)/,,$(CURDIR))
 
 # Run from container image, as recommended by https://github.com/helm/chart-testing#installation
 # `ct` analyses the Git history, so we need to mount the whole repository.
-CT := docker run --mount=type=bind,source=$(TOP_DIR),target=/src,readonly,relabel=shared --workdir=/src/$(REL_DIR) quay.io/helmpack/chart-testing:v3.5.0 ct
+CT := docker run --mount=type=bind,source=$(TOP_DIR),target=/src,readonly --workdir=/src/$(REL_DIR) quay.io/helmpack/chart-testing:v3.5.0 ct
 # Manual / local installation
 #CT := ct
 
 VERSION=$(shell cat ../../version)
 export VERSION
 
+TGZ_FILE := kuberpult-$(VERSION).tgz
+
 Chart.yaml: Chart.yaml.tpl ../../version
 	envsubst < Chart.yaml.tpl > $@
 
 values.yaml: values.yaml.tpl ../../version
 	envsubst < values.yaml.tpl > $@
 
-kuberpult-$(VERSION).tgz: Chart.yaml values.yaml templates/*
+$(TGZ_FILE): Chart.yaml values.yaml templates/*
 	helm package .
 
-ct-test:
+ct-test: Chart.yaml values.yaml
```

Comment:
You did not add targets to build `Chart.yaml` and `values.yaml` so I think this will fail

Reply:
I didn't add them, because they are already there.